### PR TITLE
#92 - Add IT for install archive with composer, add slice for download archive

### LINKED ISSUE #92: Add IT for installing uploaded archive
There is a integration test for verifying that uploaded archive could be installed with `artifacts` repository. 
It is necessary to add test that this archive could be installed using `composer` repository.
The type of repository is specified in `composer.json` file

### PR DESCRIPTION
**Pull request by genryxy**
Closes #92
Added IT to verify installation of uploaded archive through `composer` repository. In this case it generates requests to Artipie server, therefore `DownloadArchiveSlice` was added.

### CODE REVIEW

File: src/main/java/com/artipie/composer/http/DownloadArchiveSlice.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    @Override
+    public Response response(
+        final String line,
+        final Iterable<Map.Entry<String, String>> headers,
+        final Publisher<ByteBuffer> body
+    ) {
+        final RequestLineFrom rqline = new RequestLineFrom(line);
+        final String uri = rqline.uri().getPath();
+        final Matcher matcher = DownloadArchiveSlice.PATH.matcher(uri);
```
Comment (g4s8) [inline src/main/java/com/artipie/composer/http/DownloadArchiveSlice.java:72]: @genryxy the routing is an upper level operation, it's a not good idea to hide it inside `Slice` implementation. The routing slice could wrap `DownloadArchiveSlice` in top level of adapter composition, e.g.:
```java
new SliceRoute(
  new RtRule.ByPath(your_regexp_here),
  new DownloadArchiveSlice()
)
  
```

**Review by g4s8 — CHANGES_REQUESTED** [review]
@genryxy please check my comment

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

File: src/main/java/com/artipie/composer/http/DownloadArchiveSlice.java (hunk 1 above)
Comment (genryxy) [reply src/main/java/com/artipie/composer/http/DownloadArchiveSlice.java]: @g4s8 thanks, corrected. Although now mostly all slices include patterns

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#93` is not assigned, can't get performer

**Pull request merged**